pkg/sriov/sriov.go

```diff
@@ -90,8 +90,26 @@ func (s *sriovManager) SetupVF(conf *sriovtypes.NetConf, podifName string, netns
 		return fmt.Errorf("error setting temp IF name %s for %s", tempName, linkName)
 	}
 
-	// 3. Change netns
-	logging.Debug("3. Change netns",
+	// 3. Remove alt name from the nic
+	logging.Debug("3. Remove interface original name from alt names",
+		"func", "SetupVF",
+		"linkObj", linkObj,
+		"OriginalLinkName", linkName,
+		"tempName", tempName)
+	linkObj, err = s.nLink.LinkByName(tempName)
+	if err != nil {
+		return fmt.Errorf("error getting VF netdevice with name %s", tempName)
```

Comment:
wrap the original error
```suggestion
		return fmt.Errorf("error getting VF netdevice with name %s: %w", tempName, err)
```

Reply:
done thanks! :)